### ISSUE TITLE
[$10][MSFT-257] "Login" button foreground and background colors missed WCAG 2 AA contrast ratio

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2907]
**Steps to Reproduce**
1. Open private tab and go to: https://www.topcoder.com/challenges
2. Right click > Inspect Elements 
3. Check the "Login" button

**Expected Result**
- Contrast ratio should be 4.5:1
- If you want passes WCAG 2 AA, you can change text color to #0074E8. New contrast ratio: 4.5:1

**Actual Result**
Foreground Color: #0681FF
Background Color: #FFFFFF
Contrast Ratio: 3.75:1
=> Therefor, "Login" button foreground and background colors missed WCAG 2 AA contrast ratio

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.3 Contrast (minimum)
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=b9c070b1%2Dbb96%2D40e1%2Db92f%2D16f47db5d911&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fdoremihong%2FTest%20for%20Accessibility%20Challenge%2FBug%5F36

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30095947 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30095947 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by rashmi73** [comment]
@veshu @ThomasKranitsas are these open? or going to be open ?

**Comment by gurmeetb** [comment]
@rashmi73 Bug Bash is being launched, these will become open then. Keep an eye on slack challenge `dev-challenges` for Bug Bash launch.

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30095947 has been updated - it has been assigned to rashmi73.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by rashmi73** [comment]
@Oanh-and-only-Oanh ready https://github.com/topcoder-platform/community-app/pull/2953

**Comment by Dara-K** [comment]
@rashmi73 I am late to the party, but please use the #006AD7 for the button color. This will be the new blue color for all our main buttons.

**Comment by rashmi73** [comment]
@Dara-K  sure

**Comment by rashmi73** [comment]
@codeMinter  PR updated https://github.com/topcoder-platform/topcoder-react-ui-kit/pull/11

and new PR https://github.com/topcoder-platform/topcoder-react-ui-kit/pull/11

**Comment by rashmi73** [comment]
@codeMinter @sushilshinde PR for qa-accesility at topcoder-react-ui-kit
https://github.com/topcoder-platform/topcoder-react-ui-kit/pull/13

**Comment by codeMinter** [comment]
@rashmi73 let's only keep single PR, we can edit the old one for new base branch. I have updated and closed topcoder-platform/topcoder-react-ui-kit#13

**Comment by sushilshinde** [comment]
I have reverted the PR for now on community-app please PR again and test

@codeMinter @rashmi73

**Comment by codeMinter** [comment]
@tosha5252 can you add more details, what steps used where it failed? Try to add some screenshots, this looks to me like we didn't merge topcoder-platform/topcoder-react-ui-kit#11 

@sushilshinde can you take a look to confirm we merged react-ui ?

**Comment by codeMinter** [comment]
@sushilshinde updated topcoder-platform/topcoder-react-ui-kit#11 - let's merge and deploy that and then test this.

**Comment by sushilshinde** [comment]
@codeMinter https://github.com/topcoder-platform/topcoder-react-ui-kit/pull/11#issuecomment-514208837

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096722 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096722 has been updated - it has been assigned to rashmi73.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by codeMinter** [comment]
@rashmi73 please use the variable name as per - https://marvelapp.com/c7h2a6h/screen/59648056/handoff for `react-ui`

**Comment by rashmi73** [comment]
@codeMinter already there is mismatch in variables and its values in code.
because i have to use "#006AD7" for this issue and handoff has its value as "$tc-dark-blue-110".

whereas "$tc-dark-blue-110" is already taken by "#006dea" in code. then what to use?

kindly confirm

**Comment by codeMinter** [comment]
@rashmi73 kindly ignore my last comment. We will address it later.

**Comment by drasticdpk** [comment]
@rashmi73 @sushilshinde having same issue please confirm.
![image](https://user-images.githubusercontent.com/25297859/61894147-7f905580-af2f-11e9-8665-ccb1378d1c96.png)

**Comment by codeMinter** [comment]
@rashmi73 I see the issue, looks like changes done by you have got updated due to some other PR, can you create new PR and use color code as `$tc-dark-blue-110`. Kindly mark me on PR as soon as done.

**Comment by codeMinter** [comment]
> @rashmi73 I see the issue, looks like changes done by you have got updated due to some other PR, can you create new PR and use color code as `$tc-dark-blue-110`. Kindly mark me on PR as soon as done.

@rashmi73 any updates here ?

**Comment by rashmi73** [comment]
@codeMinter just 10 minutes

**Comment by rashmi73** [comment]
@codeMinter @veshu PR is now https://github.com/topcoder-platform/community-app/pull/3149

**Comment by drasticdpk** [comment]
Verified on
Operating System: Windows
Browser: Chrome

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30096722<br/><br/>```This is an automated message for crazyk via Topcoder X```
